sysbuild/Kconfig.suit

```diff
@@ -35,7 +35,7 @@ config SUIT_ENVELOPE_ROOT_TEMPLATE
 
 config SUIT_ENVELOPE_EDITABLE_TEMPLATES_LOCATION
 	string "Path to the folder with envelope templates"
-	default "../"
+	default "."
```

Comment:
I would much prefer this setting is complete removed.

Please fix the location where the file is created to the build folder, and not allow users to modify it.